### DISCUSSION TITLE
How to provide alias for schema key?

### DISCUSSION THREAD

**Discussion by AzharUddinSheikh** [react-hook-form/react-hook-form#9367] [Q&A]
I have built a `form` where my backend expecting values for `title`, `dailyRentalRate`, `genreId` etc as you can see below images label are quite descriptive (instead of = NumberInStock > Stock, DailyRentalRate > Rate etc)

<img width="654" alt="Screenshot 2022-11-14 at 7 21 09 PM" src="https://user-images.githubusercontent.com/73925826/201677160-f74a56b7-9de8-4beb-9b52-9314314ef327.png">

I have defined my schema 

<img width="581" alt="Screenshot 2022-11-14 at 7 23 52 PM" src="https://user-images.githubusercontent.com/73925826/201677506-635889f7-7d7d-482d-a18f-453d02c63ee6.png">

as my backend was expecting keys like `title, dailyRentalReate, genreId` etc that's why I had to use same name for my `key`
how could I provide `alias` for my schema key

that's why I forced to provide `label` of inputs same as `schema` key

<img width="286" alt="Screenshot 2022-11-14 at 7 29 21 PM" src="https://user-images.githubusercontent.com/73925826/201679009-b7ef4532-505e-409c-a633-51c82cd2984c.png">

**Comment by leapful** [comment]
@AzharUddinSheikh 

You can use yup Schema.label() to set another key name for error message to use (https://github.com/jquense/yup#schemalabellabel-string-schema).
